packages/workspace/src/generators/new/generate-preset.ts

```diff
@@ -16,12 +16,7 @@ import * as yargsParser from 'yargs-parser';
 import { spawn, SpawnOptions } from 'child_process';
 
 export function addPresetDependencies(host: Tree, options: NormalizedSchema) {
-  if (
-    options.preset === Preset.Apps ||
-    options.preset === Preset.Core ||
-    options.preset === Preset.Empty ||
-    options.preset === Preset.NPM
-  ) {
+  if (options.preset === Preset.Core || options.preset === Preset.Empty) {
```

Comment:
If I'm not mistaken, I think core is the same as npm and empty is the same as apps.

If so, these should also move down there. 

Separately, we should clean these up.